### DISCUSSION TITLE
Create authentification ticket in steam servers to be validate on a game server

### DISCUSSION THREAD

**Discussion by lexzor** [SteamRE/SteamKit#1336] [General]
Hello,

It's possible to use SteamKit2 to create an authentification ticket to be validated on gameserver?

Thanks!

**Comment by Hi-ImKyle** [comment]
I need this too. Hopefully it is and I'm just not seeing it.

Edit: Whether or not it's considered "shady", I have a legitimate use case for it.. Which now knowing it's classed as "shady", I guess I'll just go elsewhere.

**Comment by lexzor** [comment]
Somehow i managed to generate an authorization ticket, but not with SteamKit2.

Just use Steamworks SDK and SteamAPI with an opened Steam client. 

There is not way to init SteamAPI with SteamKit2.

**Reply by Hi-ImKyle** [reply]
I figured. Just going to use a micro service and a different lib in a different language to do it.. at least they aren't calling people shady for it.. 😒
